py3status/parse_config.py

```diff
@@ -106,6 +106,18 @@ class ConfigParser:
             }
         }
 
+    * execute shell code
+
+        order += exec(pass show git|head -n1)
```

Comment:
not sure it's worth showing a module loading based on a shell command? I'd drop this